zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -2,19 +2,11 @@
   <form class="form-inline" role="form">
     <div class="form-group">
       <select data-placeholder="Service Name" class="form-control input-sm" style="width: 234px;" name="serviceName" id="serviceName">
-        <option value=""></option>
```

Comment:
mine concern is that retrieving 100s of service names even from cache and rebuilding the DOM is expensive.